Source/WebKit/Shared/WebPreferencesDefaultValues.cpp

```diff
@@ -163,6 +163,11 @@ bool defaultCaptureAudioInUIProcessEnabled()
     return false;
 }
 
+bool defaultInvokeAttributeEnabled()
+{
+    return false;
+}
```

Comment:
You can remove this from here and the header files, since the YAML file is no longer using it.